moreDomManipulation_HW/app.js

```diff
@@ -0,0 +1,85 @@
+console.log(document.body.style.fontFamily, 'here')
+document.body.style.fontFamily = "Arial, sans-serif";
+console.log(document.body.style.fontFamily)
+document.querySelector('span#nickname').innerText = 'Timon';
+document.querySelector('span#favorite').innerText = 'Spider';
+document.querySelector('#hometown').innerText = 'El Paso';
+const listitem = document.querySelectorAll('li');
+
+for (i of listitem) {
+    i.classList.add("listitem");
```

Comment:
one for-of loop to add the class and change the color.
```suggestion
    i.classList.add("listitem");
    i.style.color = "rebeccapurple";
```